pages/events/[slug].vue

```diff
@@ -1,3 +1,248 @@
+<script setup>
+// COMPONENTS
+import { NavBreadcrumb } from 'ucla-library-website-components'
+
+// HELPERS
+import _get from 'lodash/get'
+
+// GQL
+import FTVAEventDetail from '../gql/queries/FTVAEventDetail.gql'
+
+const { $graphql } = useNuxtApp()
+
+const route = useRoute()
+
+// DATA
+const { data, error } = await useAsyncData(`events-detail-${route.params.slug}`, async () => {
+  const data = await $graphql.default.request(FTVAEventDetail, { slug: route.params.slug })
+
+  // return { data, ftvaEventSeries }
+  return data
+})
+if (error.value) {
+  throw createError({
+    ...error.value, statusMessage: 'Page not found.' + error.value, fatal: true
+  })
+}
+
+if (!data.value.ftvaEvent) {
+  // console.log('no data')
+  throw createError({
+    statusCode: 404,
+    statusMessage: 'Page Not Found',
+    fatal: true
+  })
+}
+
+// const page = ref(_get(data.value.single, 'entry', {}))
+const page = data.value.ftvaEvent
+// Breadcrumbs
+const breadcrumbs = page.title
+// Banner Header
+const imageCarousel = page.imageCarousel
+const eventTitle = page.title
+// CardMeta
+const cardMetaCategory = data.value.ftvaEventSeries[0].title
+const cardMetaTags = page.ftvaEventFilters
+const cardMetaIntro = page.ftvaEventIntroduction
+const cardMetaText = page.eventDescription
+// BlockScreening
+const screeningDetails = page.ftvaEventScreeningDetails
+// SIDEBAR
+// BlockInfo
+const blockInfo = page.ftvaTicketInformation
+
+// BlockEventDetail
+const eventDetailDate = page.startDate
+const eventDetailTime = page.startTime
+const eventDetailLocation = page.location
+
+// CardWithImage
+const series = data.value.ftvaEventSeries
+
+// const page = ref(data.value)
+watch(data, (newVal, oldVal) => {
+  console.log('In watch preview enabled, newVal, oldVal', newVal, oldVal)
+  page.value = newVal
+})
+
+</script>
+
 <template>
-  <div>This is Samople Event Detail page</div>
+  <main
+    id="main"
```

Comment:
Check this path locally /events/the-pink-cloud-shorts-03-02-24 on pnpm run dev if you get a 500 even after connecting to VPN, possibly a null or undefined issue in  this code somewhere.

Even this path /events/step-up-high-water-2018-2022-10-18-25 did not get build , on pnpm run dev and VPN, check if the page loads or gives. a 500, and fix the 500 error
 this path throws 500 error in the build /events/step-up-3-07-19-25
path /events/test-welcome-to-the-dollhouse-12-16-24 throws 500 error in build
check this route too /events/test-happiness-11-21-25

Reply:
checked them all